AUTHORS

```diff
@@ -365,3 +366,4 @@ Zach Brock <zbrock@gmail.com> <zbrock@squareup.com>
 Zachary Smith <Zachary.smith@yodle.com> Zachary.smith <Zachary.smith@yodle.com>
 何羿宏 <heyihong.cn@gmail.com>
 智雅楠 <zac.zhiyanan@gmail.com>
+
```

Comment:
Would you mind removing the new line at the end of the file? We like to only change the lines that need changing, which in this case is adding your name to the list.